content/influxdb/v2.0/influxdb-templates/_index.md

```diff
@@ -22,6 +22,8 @@ share your configuration with the InfluxData community.
 
 {{< youtube 2JjW4Rym9XE >}}
 
+[InfluxData Community Templates](https://github.com/influxdata/community-templates/)
```

Comment:
This is a pretty significant feature for new users, so I'm wondering if we bump up the hierarchy of this link by formatting it as a button and a call to action.

```suggestion
<a class="btn github" href="https://github.com/influxdata/community-templates/" target="_blank">View InfluxDB community templates</a>
```

Side note – I need to create a shortcode for creating buttons 😄

Reply:
Thanks! I was thinking the same thing, but wasn't sure whether it qualified.